.github/dependabot.yml

```diff
@@ -11,3 +11,11 @@ updates:
     schedule:
       # Check for updates to GitHub Actions every week
       interval: "weekly"
+    commit-message:
+      prefix: "BAU"
+  - package-ecosystem: docker
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: BAU
```

Comment:
See previous comment; we can't codify practices that we don't officially endorse.